bin/Makefile.am

```diff
@@ -33,15 +33,14 @@ dist_bin_SCRIPTS = ch-build \
 
 ## Python scripts - need text processing
 
-bin_SCRIPTS = ch-run-oci          # scripts to build
-EXTRA_SCRIPTS = ch-image ch-grow  # more scripts that *may* be built
+bin_SCRIPTS = ch-run-oci  # scripts to build
```

Comment:
Are we shoveling `ch-run-oci` too?

Reply:
No; it'll stay. We just don't CI it currently.